src/gameObjects/map_start/misterFu.gameObject.js

```diff
@@ -0,0 +1,33 @@
+import { scaleFactor } from '../../constants';
+
+export const misterFu = (k, map, spawnpoints) => {
+    k.loadSprite('misterFu', './misterFu.png', {
```

Comment:
unfortunately, there is no misterFu.png available. Please use the characters.png from public instead. The coordinates change in this case as well. Have a look at src/constants.js to get the coordinates for misterFu